recipes/opencolorio/all/patches/2.3.1-0001-fix-cmake-source-dir-and-targets.patch

```diff
@@ -59,9 +59,18 @@ index 175d89c2..ac93b87a 100644
  include(CheckCXXCompilerFlag)
  
 diff --git src/OpenColorIO/CMakeLists.txt src/OpenColorIO/CMakeLists.txt
-index 26b4bb4c..d54cff3d 100755
+index 26b4bb4c..f366bdd1 100755
 --- src/OpenColorIO/CMakeLists.txt
 +++ src/OpenColorIO/CMakeLists.txt
+@@ -301,7 +301,7 @@ target_include_directories(OpenColorIO
+ )
+ 
+ target_link_libraries(OpenColorIO
+-    PRIVATE
++    PUBLIC   # Change to public so that the tools properly link
```

Comment:
Has this been submitted upstream? This looks like something that would better live as an official change upstream, where it can be properly maintained

Reply:
I'm going to put up a pull request, but I'm currently working on getting into the CLA for the Linux Foundation which the upstream is part of. But I can also start the pull request later without the CLA standing and retrigger the check later. (And then reference the PR in the conandata)

Reply:
@irieger If you have a blocker with CLA because your job (for instance), don't worry, we can send the patch too. Thank you for checking it!

Reply:
No, the job is not a problem. It is more the other way round. Trying to get the CLA through my company (we have a company CLA there) as I don't like the Linux foundation requireing a physical address, which is why I refuse to sign it with my personal details. (An open source focused organisation like that having such a bad privacy statement is a red flag for me when asked for personal details.)

Me doing this is no problem for my company and although this contributions here are for my personal interest, I guess it is good to be active in open source projects from a company perspective anyway and I have some colleagues contributing to the Academy Software Foundation projects (ASFW is hosted - or whatever the right english word is here - by the Linux Foundation).

Reply:
Created a PR: https://github.com/AcademySoftwareFoundation/OpenColorIO/pull/1958

Reply:
As this is anyway only a small part of the overall compatibility conan patching, I guess it doesn't make sense to add the upstream URL maybe?